src/scss/overrides/_type.scss

```diff
@@ -1,191 +1,49 @@
-h1 {
-  margin: 20px 0;
-  font-size: 3rem;
-  line-height: 1;
-  letter-spacing: -0.1rem;
-  font-weight: 400;
-  color: $charcoal;
-  @include media-breakpoint-up(sm) {
-    line-height: 1.2;
-  }
-}
-
-.header-headline {
-  margin-bottom: 30px;
-  padding: 15px 0 10px 0;
-  border-bottom-width: 2px;
-  border-bottom-style: solid;
-}
-
-h2 {
-  margin: 20px 0 10px 0;
-  font-size: 2.8rem;
-  line-height: 1;
-  letter-spacing: -0.1rem;
-  font-weight: 200;
-  color: $charcoal;
-  @include media-breakpoint-up(sm) {
-    line-height: 1.2;
+h1, h2, h3, h4, h5, h6,
+.h1, .h2, .h3, .h4, .h5, .h6 {
+  &:not(:first-child) {
+    margin-top: 1em;
```

Comment:
I'm not too keen on this rule. Usually in this case the previous sibling should have bottom margin on it.